code/game/machinery/computer/chef_orders/chef_order.dm

```diff
@@ -0,0 +1,126 @@
+
+/obj/machinery/computer/chef_order
+	name = "Produce Orders Console"
+	desc = "An interface for ordering fresh produce and other. A far more expensive option than the botanists, but oh well."
+	icon_screen = "request"
+	icon_keyboard = "generic_key"
+	circuit = /obj/item/circuitboard/computer/chef_order
+	light_color = LIGHT_COLOR_ORANGE
+
+	COOLDOWN_DECLARE(order_cooldown)
+	var/list/order_datums = list()
```

Comment:
Are you sure this should not be a static list? Is it different across computers?